docs/kb/release-notes.md

```diff
@@ -0,0 +1,29 @@
+# Release Notes
+
+Here you'll find the latest updates on what's changed in the Pardot API and API documentation.
+
+## March 2017
+
+### Visitor Activities 
+
+A few updates were made to the `query` endpoint for both version 3 and 4. These include:
```

Comment:
No need for flourishing with superfluous 'a few updates' or 'these include. EX:
#### 'query' end-point (v3-4):
* blah
* blah 

also, use end-point or end point (Endpoint is a proper noun for several companies, band names, etc.)